# Generated from: /home/latty/link_dir/2015_li3ds/__ROS__/Ellipse_N/ROS/overlay_ws/src/sbg_ros_driver/sbgECom/src/binaryLogs/sbgEComBinaryLogShipMotion.h
# SbgLogShipMotionData
uint32 timeStamp
uint16 status
float32 mainHeavePeriod
float32[3] shipMotion
float32[3] shipAccel
float32[3] shipVel